works.html

```diff
@@ -0,0 +1,103 @@
+    <div class="lang-4">
+    <p class="css">css</p>
+    <p class="html">html</p>
+    <p class="bootstrap">bootstrap</p>
+    <p class="Ruby">Ruby</p>
+  </div>
+    <p class="see-project">See Project</p>
+  </section>
+
+  <section class="projects">
+    <div class="project project1">
+      <h2 class="professional">Professional Art Printing Data</h2>
+      <p class="proj-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <div class="lang-3">
+
+        <ul class="buttons">
+          <li><a href="#">html</a></li>
+          <li><a href="#">bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+
+      </div>
+      <p class="project-link"><a href="#">See Project</a></p>
+    </div>
+    <div class="project project1">
+      <h2 class="professional">Professional Art Printing Data</h2>
+      <p class="proj-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <div class="lang-3">
+
+        <ul class="buttons">
+          <li><a href="#">html</a></li>
+          <li><a href="#">bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+
+      </div>
+      <p class="project-link"><a href="#">See Project</a></p>
+    </div>
+    <div class="project project1">
+      <h2 class="professional">Professional Art Printing Data</h2>
+      <p class="proj-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <div class="lang-3">
+
+        <ul class="buttons">
+          <li><a href="#">html</a></li>
+          <li><a href="#">bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+
+      </div>
+      <p class="project-link"><a href="#">See Project</a></p>
+    </div>
+    <div class="project project1">
+      <h2 class="professional">Professional Art Printing Data</h2>
+      <p class="proj-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <div class="lang-3">
+
+        <ul class="buttons">
+          <li><a href="#">html</a></li>
+          <li><a href="#">bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+
+      </div>
+      <p class="project-link"><a href="#">See Project</a></p> 
+    </div>
+    <div class="project project1">
+      <h2 class="professional">Professional Art Printing Data</h2>
+      <p class="proj-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <div class="lang-3">
+
+        <ul class="buttons">
+          <li><a href="#">html</a></li>
+          <li><a href="#">bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+
+      </div>
+      <p class="project-link"><a href="#">See Project</a></p>
```

Comment:
- [ ] The button `See project` is supposed to be a link `a` or `button` tag for semantic reasons, you don't have to grab it in a paragraph since it's not a text paragraph, and follow the button styling: `Active`, `Hover` and `Pressed` as shown on the design in Figma